pyk/src/pyk/kast/outer_parser.py

```diff
@@ -120,8 +125,17 @@ def module(self) -> Module:
         while self._la.type is not TokenType.KW_ENDMODULE:
             sentences.append(self.sentence())
 
+        end_loc = self._la.loc + self._la.text
         self._consume()
 
+        location = (
```

Comment:
Consider making this an attribute of AST elements instead. That also makes parsing location attribute strings unnecessary.